lighthouse-core/report/html/templates.html

```diff
@@ -262,6 +262,14 @@
       height: var(--lh-tools-icon-size);
       cursor: pointer;
       margin-right: 5px;
+      /* Override default button styles. */
```

Comment:
maybe add to the comment that this is an actual `button` element but we want to style it like a transparent `div`? :)